### PR TITLE
initial commit from local branch

### LINKED ISSUE #15: wordsoup workflow progress
# Workflow
- [x] Create a branch
- [x] Make commits on new branch
- [x] Open a pull request
- [x] Collaborate :tada: 
- [x] Make more commits
- [x] Merge into master

:tea: :chocolate_bar:

### PR DESCRIPTION
**Pull request by wordsoup**
This pull request closes #15 

Initial pull request adding wordsoup hometown info

### CODE REVIEW

File: wordsoup-longmont.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# Longmont
+
+### Things to Do
+- Dickens Tavern
```
Comment (wordsoup) [inline wordsoup-longmont.md:4]: test PR comment

Comment (wordsoup) [inline wordsoup-longmont.md:4]: Test 2

**Comment by joshualester2112** [comment]
@wordsoup are there any other good watering holes in Longmont❔

**Comment by wordsoup** [comment]
Lefthand Brewery and Oskar Blues are both headquartered here, which are great!

**Pull request merged**